doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -3401,12 +3503,26 @@ Build system
 
 The issues in this section are related to :ref:`app_build_system`.
 
-.. rst-class:: v2-9-0-nRF54H20-1 v2-9-1 v2-9-0 v2-8-0
+.. rst-class:: v3-0-0
+
+VSC-2974: On Windows, some sample paths are too long
+  The maximum full path to an object file is 250 characters.
+
+  **Workaround:** Symlink your development sample folder to a shorter path.
+  For example:
+
+  .. code-block::
+
+     mklink  /J  C:\ncs\sample-dev C:\ncs\v3.0.0\nrf\samples\cellular\http_update\modem_full_update
+
+  Import :file:`C:\ncs\sample-dev` to VS-Code, and the build should work.
+
+.. rst-class:: v3-0-0 v2-9-0-nRF54H20-1 v2-9-1 v2-9-0 v2-8-0
 
 NCSDK-30119: For nRF54L15 SoC, dynamic partitioning for a project building with MCUboot's direct-xip mode is not supported
   Static partition manager file is required for building such project.
 
-.. rst-class:: v2-9-0-nRF54H20-1 v2-9-1 v2-9-0 v2-8-0 v2-7-0
+.. rst-class:: v3-0-0 v2-9-0-nRF54H20-1 v2-9-1 v2-9-0 v2-8-0 v2-7-0
 
 NCSDK-28495: Sysbuild runs CMake code before processing application :file:`sysbuild.cmake`
```

Comment:
this was resolved here: https://github.com/nrfconnect/sdk-nrf/pull/16576
fix is in v2.8.0